ui/imports/shared/popups/keycard/states/KeycardInit.qml

```diff
@@ -775,7 +762,7 @@ Item {
                 startImgIndexForTheFirstLoop: 0
                 startImgIndexForOtherLoops: 0
                 endImgIndex: 20
-                duration: 1000
+                duration: 1300
```

Comment:
Wondering why

Reply:
In all other places where we're using `strong_success` animation the length of a single animation is set to 1300 ms, and wanted to have that everywhere the same, but in general we can adjust that later. In the next PR will set all those durations as constants, will be easier for maintaining.